apps/visited/src/lib.rs

```diff
@@ -0,0 +1,52 @@
+#![allow(clippy::len_without_is_empty)]
+
+use std::result::Result;
+
+use calimero_sdk::app;
+use calimero_sdk::types::Error;
+use calimero_storage::collections::{UnorderedMap, UnorderedSet};
+use calimero_storage::entities::Element;
+use calimero_storage::AtomicUnit;
+
+#[app::state]
+#[derive(AtomicUnit, Clone, Debug, PartialEq, PartialOrd)]
+#[root]
+#[type_id(1)]
+pub struct VisitedCities {
+    visited: UnorderedMap<String, UnorderedSet<String>>,
+    #[storage]
+    storage: Element,
+}
+
+#[app::logic]
+impl VisitedCities {
+    #[app::init]
+    pub fn init() -> VisitedCities {
+        VisitedCities {
+            visited: UnorderedMap::new().unwrap(),
+            storage: Element::root(),
+        }
+    }
+
+    pub fn add_person(&mut self, person: String) -> Result<bool, Error> {
+        Ok(self.visited.insert(person, UnorderedSet::new().unwrap())?)
```

Comment:
avoid `unwrap` everywhere but `init`, which has technical limitations for why we can't return a result

on perhaps we make it so `UnorderedMap::new` doesn't return an error, and panics internally if it can't be created

Reply:
will fix in follow up PR this is just an example to get master int o good state